crates/api/src/site/registration_applications/list.rs

```diff
@@ -1,4 +1,5 @@
-use actix_web::web::{Data, Json, Query};
+use activitypub_federation::config::Data;
+use actix_web::web::{Json, Query};
```

Comment:
All these import changes seem unnecessary.

Reply:
they're necessary to be able to pass the test context returned by this, as that is returning `activitypub_federation::config::Data`, not `actix_web::web::Data`:

https://github.com/LemmyNet/lemmy/blob/a08642f81377e8ef01f91f5d49f531e71fd62d70/crates/api_common/src/context.rs#L84-L87